package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "dev": "PORT=8080 next dev",
     "build": "next build",
+    "debug:standalone": "next build && cp -rf public .next/standalone/ && cp -rf .next/static .next/standalone/.next/static && cp -rf node_modules/web-worker/cjs/browser.js .next/standalone/node_modules/web-worker/cjs/ && node .next/standalone/server.js",
```

Comment:
these files ares are required to be copied `public` & `static`.
the `web-worker` copy is strange but needed for `monero-ts`seems like the browser.js isnt copied prorperly